### ISSUE TITLE
Unstable Results Over 500 Round Experiments

### ISSUE THREAD

**Issue by AndrewMerrow** [TinfoilHat0/Defending-Against-Backdoors-with-Robust-Learning-Rate#4]
I have tried running tests on the fedemnist dataset with the default parameters from the runner.sh file.  In my 500 round tests, the model's accuracy starts to degrade after approximately round 100.

![UTD 500 round graph](https://github.com/TinfoilHat0/Defending-Against-Backdoors-with-Robust-Learning-Rate/assets/48524719/be8472c3-2738-4bbc-a622-0cf23769a8ae)

I have ran experiments in two separate environments and have tried tweaking some parameters, but the results I am getting all show the same issue.  Here are the library versions I am using:

NVIDIA PyTorch Container version 22.12
PyTorch version 1.14.0+410ce96
Python3 version 3.8.10

**Comment by TinfoilHat0** [comment]
Hey Andrew - the plot makes me this this a learning rate problem. Are you decaying the learning rate?

**Comment by AndrewMerrow** [comment]
I have not decayed the learning rate. Here are the values I have used:
server_lr: 1
client_lr: 0.1

**Comment by AndrewMerrow** [comment]
We used all the parameters straight from the provided runner.sh file.
